src/nginxconfig/util/import_data.js

```diff
@@ -102,6 +102,15 @@ export default (query, domains, global, nextTick) => new Promise(resolve => {
             // Create a new domain (assume it has had custom user settings)
             const domainImported = clone(Domain.delegated);
             domainImported.hasUserInteraction = true;
+
+            //  Backwards compatibility logic for old config URLs
+            if (isObject(data.global.php)){
+                if(data.global.php.phpServer)  domainImported.php.phpServer = data.global.php.phpServer;
+                if(data.global.php.phpServerCustom)  domainImported.php.phpServerCustom = data.global.php.phpServerCustom;
+                if(data.global.php.phpBackupServer)  domainImported.php.phpBackupServer = data.global.php.phpBackupServer;
+                if(data.global.php.phpBackupServerCustom)  domainImported.php.phpBackupServerCustom = data.global.php.phpBackupServerCustom;
+            }
```

Comment:
This should be moved to the backwards compatibility file, as it is backwards compatibility.

The best thing to do isto refactor the backwards compatibility file, renaming the current function in there to be `angularBackwardsCompatibility`, and then move this logic into a new function called `vueBackwardsCompatibility` -- it is important that we keep the angular backwards compatibility logic separate from any new backwards compatibility logic, but they should both be part of the backwards compatibility file. The exported function can then apply both functions.

Reply:
What do you think of this workaround bellow?  :sweat_smile:
[Add here](https://github.com/digitalocean/nginxconfig.io/blob/4cfdcdb8d1c2df185e9ba437ca9c79e7c2ef17e8/src/nginxconfig/util/import_data.js#L114)
```javascript
// Handle converting vue params to the current version
vueBackwardsCompatibility(data, domains);
```
In: `vue_backwards_compatibility.js`
```javascript 
export default (data, domains) => {
    // Handle converting domain settings
    if ('global' in data && isObject(data.global)) {
        for (const key in data.global) {
               for (const index in domains) {
                // Handle converting old global settings to new ones
                if (isObject(data.global.php)){
                    if(data.global.php.phpServer)  domains[index].php.phpServer.value = data.global.php.phpServer;
                    if(data.global.php.phpServerCustom)  domains[index].php.phpServerCustom.value = data.global.php.phpServerCustom;
                    if(data.global.php.phpBackupServer)  domains[index].php.phpBackupServer.value = data.global.php.phpBackupServer;
                    if(data.global.php.phpBackupServerCustom)  domains[index].php.phpBackupServerCustom.value = data.global.php.phpBackupServerCustom;
                }
            }
        }
    }
};
```

Reply:
Yeah, that seems fine to me -- we should rename `backwardsCompatibility` to `angularBackwardsCompatibility` at the same time to be clear :) Ty!

Reply:
All right I did it too. :bowtie: